### ISSUE TITLE
A11y bug: Focus rings on How To pages

### ISSUE THREAD

**Issue by machikoyasuda** [cal-itp/mobility-marketplace#670]
These issues came up while I was reviewing the new Remix page, and I double-checked that these issues were present on production even before the Remix page was developed.

## Issues
- Critical issue: The focus ring for buttons and links on a dark blue background aren't visible, b/c the focus-visible is set to be _transparent_
- Minor issue: The focus ring on the side navigation links are getting cut off on the left-hand side.

## How to fix
- Focus ring not visible: Make a focus ring design for buttons on this dark background, and set it here on this CSS Variable:

```css
.btn:focus-visible {
  --bs-btn-focus-box-shadow: 0 0 0 2px #2ea3f2;
}
```

Current: Transparent!
<img width="437" height="121" alt="Image" src="https://github.com/user-attachments/assets/dd220021-05a4-4733-b41d-161af2742dcb" />

- Focus ring cut off: Change the outline to a border, so the line gets created inside the DOM element, not just outside of it

```css
  .toc-container {
    a {
      text-decoration: none !important;
      font-weight: 500;
    }
    a:focus {
      border: 2px solid #2ea3f2;
      outline: none;
    }
  }
```
